.github/workflows/publish.yml

```diff
@@ -29,13 +29,13 @@ jobs:
       run: |
         cd ./stubs && python3 setup.py sdist bdist_wheel --universal && cd -
     - name: Publish metaflow-stubs package
-      uses: pypa/gh-action-pypi-publish@9b8e7336db3f96a2939a3e9fa827c62f466ca60d # master
```

Comment:
- I think we just need to do release/v1 according to the doc

Reply:
using hash for `v1.8.11`